lms/djangoapps/courseware/entrance_exams.py

```diff
@@ -43,118 +43,16 @@ def user_can_skip_entrance_exam(user, course):
     return False
 
 
-def user_has_passed_entrance_exam(request, course):
+def user_has_passed_entrance_exam(user, course):
     """
     Checks to see if the user has attained a sufficient score to pass the exam
     Begin by short-circuiting if the course does not have an entrance exam
     """
     if not course_has_entrance_exam(course):
         return True
-    if not request.user.is_authenticated():
-        return False
-    entrance_exam_score = get_entrance_exam_score(request, course)
-    if entrance_exam_score >= course.entrance_exam_minimum_score_pct:
-        return True
-    return False
-
-
-# pylint: disable=invalid-name
-def user_must_complete_entrance_exam(request, user, course):
```

Comment:
**Note to reviewers:** The `user_must_complete_entrance_exam` method was removed since it's now simply a negation of `user_can_skip_entrance_exam`.